scripts/before_ci_build.sh

```diff
@@ -7,7 +7,12 @@ if [ ! -f finish_before_ci_build ]; then
   if [[ "$OSTYPE" == "linux-gnu" || "$OSTYPE" == "linux-musl" || "$OSTYPE" == "darwin"* ]]; then
     curl -s -O https://ftp.gnu.org/gnu/gmp/gmp-${GMP_VERSION}.tar.xz
     tar -xf gmp-${GMP_VERSION}.tar.xz
-    cd gmp-${GMP_VERSION} && ./configure --enable-fat && make -j4 && make install && cd ../
+    cd gmp-${GMP_VERSION}
+    # config.guess uses microarchitecture and configfsf.guess doesn't
+    # We replace config.guess with configfsf.guess to avoid microarchitecture
+    # specific code in common code.
+    rm config.guess && mv configfsf.guess config.guess && chmod +x config.guess
+    ./configure --enable-fat && make -j4 && make install && cd ../
```

Comment:
I think you could use just ``--build=$(./configfsf.guess)``.  But anyway, this workaround probably should be restored.